refactor(coverage/typescript): Use regex to capture `TSxxxx` error codes

leaysgur opened the pull request:
Part of #11582 

I found that consecutive `CRLF`s can appear anywhere, so changing to the robust regex just in case.

Boshen commented:
### Merge activity

* **Jun 27, 9:33 AM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

leaysgur committed:
refactor(coverage/typescript): Use regex to capture `TSxxxx` error codes (#11932)

Part of #11582

I found that consecutive `CRLF`s can appear anywhere, so changing to the robust regex just in case.
